MediaBrowser.Model/Dto/PlaylistDto.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using MediaBrowser.Model.Entities;
+
+namespace MediaBrowser.Model.Dto;
+
+/// <summary>
+/// DTO for playlists.
+/// </summary>
+public class PlaylistDto
+{
+    /// <summary>
+    /// Gets or sets a value indicating whether the playlist is publicly readable.
+    /// </summary>
+    public bool OpenAccess { get; set; }
+
+    /// <summary>
+    /// Gets or sets the share permissions.
+    /// </summary>
+    public IReadOnlyList<PlaylistUserPermissions>? Shares { get; set; }
```

Comment:
What is the difference between null and empty list here? (Same question for the ItemIds)

Reply:
I think the properties should just be marked as `public required` and nullability removed